packages/api/src/command/hie/unlink-patient-from-organization.ts

```diff
@@ -161,10 +161,16 @@ function getDocumentsWithOid(
     const patient = document.contained?.find(isPatient);
     if (!patient) continue;
 
-    const identifier = patient.identifier?.find(identifier => identifier.system === urnOid);
+    const identifier = patient.identifier?.find(identifier => identifier.system?.includes(urnOid));
     if (identifier) {
       matchingDocumentRefs.push(document);
     }
+
+    const masterIdentifier = document.masterIdentifier?.value?.includes(oid);
```

Comment:
There was a case where doc ref didnt have a patient.identifier but it was in master identifier